tests/utests/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(local_tests
+    utest:test_common
+    utest:test_set
+    utest:test_hash_table
+    utest:test_context
+    utest:test_xml
+    utest:schema/test_schema
+    utest:schema/test_parser_yang
+    utest:schema/test_parser_yin
+    utest:schema/test_tree_schema_compile
+    utest:schema/test_printer_yang
+    utest:schema/test_printer_yin
+    utest:data/test_tree_data
+    utest:data/test_parser_xml
+    utest:data/test_printer_xml
+    utest:data/test_validation
+    utest:data/test_types
+    utest:extensions/test_metadata
+    utest:extensions/test_nacm)
+set(local_tests_wraps
+    "-Wl,--wrap=realloc"
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " "
+    " ")
+set(tests ${tests} ${local_tests} PARENT_SCOPE)
+set(tests_wraps ${tests_wraps} ${local_tests_wraps} PARENT_SCOPE)
```

Comment:
We could change this so that all the tests do not have to be explicitly mentioned, just use a `*.c` regex instead, it makes adding/removing tests transparent. However, the one test with a linker wrap will need to be handled specially and excluded from the regex. I still think it is worth it but it is up to a discussion.

Reply:
It will require complete redesign of the build process. I'm not sure it is worth it. 

My idea here is that the lines do not correspond to the test source files, but to the test processes. For example `utest:schema/test_schema` already includes tests from `tests/utests/schema/test_schema_common.c` and `tests/utests/schema/test_schema_stmts.c`. This way we can split source files to keep them reasonable big/small while keeping relatively small number of the test processes.

Reply:
Well, it is not possible if several sources will be compiled into a single executable test. If that is the definite plan, then I guess it must stay this way.